alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/executors/TikaJavaExecutor.java

```diff
@@ -67,7 +65,9 @@
     
     private static final String ID = "tika";
 
-    public static final String LICENCE = "This transformer uses Tika from Apache. See the license at http://www.apache.org/licenses/LICENSE-2.0. or in /Apache\\ 2.0.txt";
+    public static final String LICENCE = 
+        "This transformer uses Tika from Apache. See the license at http://www.apache.org/licenses/LICENSE-2.0. or in /Apache\\ 2.0.txt\n" +
+        "This transformer uses ExifTool by Phil Harvey. See license at https://exiftool.org/#license. or in /Perl GNU License.txt & /Perl Artistic License.txt";
```

Comment:
As above,  link to exiftool + Perl Artistic license only.

Reply:
GNU licences removed.